webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -0,0 +1,24 @@
+def calculate_pauschbetrag(pflegegrad=False, beh_grad=None, merkzeichen_bl=False, merkzeichen_tbl=False, merkzeichen_h=False,
+                           merkzeichen_ag=False, merkzeichen_g=False):
```

Comment:
> Do you think we should omit the not needed merkzeichen in the function declaration? I have added them to clarify what we need but they do not make any difference for the result

I completely get why you included them and I think I'd actually prefer them being there..*but* at some point we're gonna delete them (if not done automatically) and also: if they're not needed, we'd need to test that setting them doesn't change anything and that creates unnecessary test cases that we can avoid by just not being able to set them here.